d2l/model.py

```diff
@@ -140,16 +140,16 @@ def forward(self, X, *args):
 
 def masked_softmax(X, valid_length):
     if valid_length is None:
-        return X.softmax()
+        return npx.softmax(X)
     else:
         shape = X.shape
         if valid_length.ndim == 1:
             valid_length = valid_length.repeat(shape[1], axis=0)
         else:
             valid_length = valid_length.reshape((-1,))
-        X = nd.SequenceMask(X.reshape((-1, shape[-1])), valid_length, True,
```

Comment:
we'd better rename it to `sequence_mask` to match the naming convention.

Reply:
Will do.

Reply:
I will submit a PR dedicated for fixing all the nn op names after this is merged.